boogiestats/templates/boogie_ui/scorecard.html

```diff
@@ -0,0 +1,172 @@
+{% load mathfilters %}
+<div class="text-center">
+    <canvas id="statcard" width="400" height="600"></canvas>
+</div>
+<script>
+    function getLines(ctx, text, maxWidth) {
+        let words = text.split(" ");
+        let lines = [];
+        let currentLine = words[0];
+
+        for (let i = 1; i < words.length; i++) {
+            let word = words[i];
+            let width = ctx.measureText(currentLine + " " + word).width;
+            if (width < maxWidth) {
+                currentLine += " " + word;
+            } else {
+                lines.push(currentLine);
+                currentLine = word;
+            }
+        }
+        lines.push(currentLine);
+        return lines;
+    }
+
+    const canvas = document.getElementById('statcard')
+    const context = canvas.getContext('2d')
+    const nodec = '{{ score.comment }}'.includes('No Dec/WO')
```

Comment:
this is unreliable. We could just display 0s for them (it's saved as 0 in the DB in that case anyway). BTW, there's also option to disable just WO ;)

Reply:
Left this on purpose as this comment is not that uncommon to gray it out. On every other outcome it's just 0 and working properly